pkg/util/mocks/env/core.go

```diff
@@ -8,12 +8,13 @@ import (
 	context "context"
 	reflect "reflect"
 
-	env "github.com/Azure/ARO-RP/pkg/env"
-	azureclient "github.com/Azure/ARO-RP/pkg/util/azureclient"
-	liveconfig "github.com/Azure/ARO-RP/pkg/util/liveconfig"
 	azcore "github.com/Azure/azure-sdk-for-go/sdk/azcore"
 	autorest "github.com/Azure/go-autorest/autorest"
 	gomock "github.com/golang/mock/gomock"
+
```

Comment:
No need to make this change - we're not touching this code in this PR.

Reply:
fixed